opencanada/settings/base.py

```diff
@@ -188,6 +188,7 @@ def get_env_variable(var_name):
 #     },
 # }
 
+WAGTAILSEARCH_RESULTS_TEMPLATE = 'basic_site/search_results.html'
```

Comment:
Since you are overriding it anyway, you could have picked a different path if you wanted (like core/search_results.html), but this will work too.  It will default back to the on in basic_site if ours disappears for some reason.